### ISSUE TITLE
'Range.detach' is now a no-op, as per DOM (http://dom.spec.whatwg.org/#dom-range-detach). 

### ISSUE THREAD

**Issue by ernestorocha** [nate-strauser/meteor-x-editable-bootstrap#24]
I believe that the version of Rangy used by this project is causing this warning in Chrome as you can see here https://github.com/timdown/rangy/issues/223.

Warning:

```
'Range.detach' is now a no-op, as per DOM (http://dom.spec.whatwg.org/#dom-range-detach). 
```

Can you update Rangy, please ?

**Comment by ndemoreau** [comment]
I could be wrong but I think that this issue is coming from wysihtml5. It has been fixed in version 0.3.3 but you are using 0.3.0.

Look at this: https://github.com/Waxolunist/bootstrap3-wysihtml5-bower/issues/88

**Comment by achirkof** [comment]
Hello. Thank you for the package. 
About issue. This package is build with wysihtml5-0.3.0 and bug is related to this version. Now actual version is wysihtml5-0.3.3 without bug. Is it possible to rebuild package with new version?
